src/components/atoms/BooleanCell/BooleanCell.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+type BooleanCellProps = {
+    booleanVal?: boolean;
+    onOffVals: {
+        on: string,
+        off: string
+    }
+}
+
+const BooleanCell: React.FC<BooleanCellProps> = ({ booleanVal = false, onOffVals }) => {
+    return (
+        <td>{booleanVal ? onOffVals.on : onOffVals.off}</td>
+    );
+}
```

Comment:
This is an interesting solution. What are your thoughts on using the ternary expression as you have done from the parent to decide on what to render within the cell? I'll comment on the table component with an example.

Reply:
I agree, we should hoist the conditional logic to the parent and try to keep the child components as simple as possible.